scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -197,6 +198,10 @@ class Job(val args: Args) extends FieldConversions with java.io.Serializable {
       .toMap.toMap[AnyRef, AnyRef] // linter:ignore the second one is to lift from String -> AnyRef
   }
 
+  def reflectedClasses: Set[Class[_]] = if (args.boolean("scalding.nojobclassreflection")) Set.empty else {
```

Comment:
How about we add it to the companion Args object? We don't need to move refactor the other Args in this review but it seems nicer than having them scattered in a bunch of places right?